src/ipc/ipc.c

```diff
@@ -436,49 +436,36 @@ int ipc_comp_dai_config(struct ipc *ipc, struct sof_ipc_dai_config *config)
 	struct sof_ipc_comp_dai *dai;
```

Comment:
s/lover/lower. There are still 3 months until Valentines day :)

Reply:
It will be soon :)